MekHQ/src/mekhq/campaign/mission/atb/scenario/BaseAttackBuiltInScenario.java

```diff
@@ -80,7 +80,7 @@ public void setExtraMissionForces(Campaign campaign, ArrayList<Entity> allyEntit
 
 		// the "second" force will be deployed (orthogonally) between 90 degrees clockwise and counterclockwise from the "primary force".
 		int angleChange = Compute.randomInt(2) - 1;
```

Comment:
I'm not sure the original code is correct here. `Compute.randomInt(2)` will produce either 0 or 1, so `angleChange` is -1 or 0, and the second attacker is given a start position that is either 90 anticlockwise or the same as the first. The change shifts it to clockwise instead if the first attacker position is north instead of throwing an exception.